### ISSUE TITLE
Optionally show y-axis and x-axis values outside of graph

### ISSUE THREAD

**Issue by JoryHogeveen** [kalkih/mini-graph-card#739]
Just like the core statistics graph it would be nice to be able to show the axis values outside of the graph.
Things like time etc. would be more much clear :)

https://www.home-assistant.io/lovelace/statistics-graph/

**Comment by jlsjonas** [comment]
This feature sadly enough doesn't fit well with the design language we have in mind for the card, unless I misunderstand your request. Could you give a more concrete example that would need this?
Thanks for your contribution!

**Comment by JoryHogeveen** [comment]
Hi @jlsjonas 

What do you mean with that it wouldn't fint in the current design langauge?
This would be an extra for the graph (below or beside it), not a modification.

## Examples:

**Current:**
![image](https://user-images.githubusercontent.com/826148/157038389-a1fb7544-28de-4435-8c06-16e86ccc8d1f.png)

**With Y-axis as days/time (see below):**
![image](https://user-images.githubusercontent.com/826148/157040177-315fcd24-d69d-45cd-8def-6a3f074ed990.png)

**Comment by jlsjonas** [comment]
The principles behind mini graph card is to be quite simple & clean, however I can see enough benefit towards considering at least x axis to provide better user feedback.

**Comment by Grey-Lancaster** [comment]
I would love x and y-axis also :-)

![image](https://user-images.githubusercontent.com/20055899/171040404-fae86919-febc-4597-80d4-3cd4d0830478.png)

**Comment by xthursdayx** [comment]
I would also love this feature, it's the main thing that causes me to go to ApexCharts rather than mini-graph-card.

**Comment by JoryHogeveen** [comment]
Just as an update for others seeking the same feature.
I've completely switched to apex cards which has x/y axis management and much much more features :)
Hopy this helps!

**Comment by teklabania** [comment]
Apex is ok, but is much slower/heavier then mini-graph-card.

**Comment by JoryHogeveen** [comment]
> Apex is ok, but is much slower/heavier then mini-graph-card.

Didn't have such issues with it tbh. Then again, because it can support many things I can imagine that based on your config it could get heavier on your system, which is only logical.

**Comment by Brandon1212b** [comment]
Any update? displaying x-axis seems like a required feature with any graph

**Comment by MiCRoPhoBIC** [comment]
This is indeed a serious drawback

**Comment by douginoz** [comment]
Clearly, the authors think that analytical charts are art and not intended to be informative and used by those that specifically need data to be displayed in a chart.
Why spoil data by making it useful?  More colours!  Flowers!  Built-in soundtrack!

Idiots.

**Comment by ildar170975** [comment]
> Clearly, the authors think that analytical charts are art and not intended to be informative and used by those that specifically need data to be displayed in a chart. Why spoil data by making it useful? More colours! Flowers! Built-in soundtrack!
> 
> Idiots.

I intentionally not deleted your inappropriate post. 
Since you probably consider yourself as “not idiot” - your options are:
1. Stop using this FREE software and start using another one which you like more. 
2. Contribute to this project which definitely lacks developers since the original author cannot maintain the project due to serious reasons. Warning: as a contributor, you will probably be called an “idiot” as well one lucky day - if someone like you starts talking about “what a bullshit you made”.

**Comment by zunami** [comment]
**Body:**
Hello @kalkih and community,

I love using the `mini-graph-card` for its clean, lightweight charts. However, one limitation I'm missing is the ability to display **weekday labels (Mon, Tue, Wed, etc.)** along the X-axis. This would be particularly useful for _weekly_ and _multi-day_ views, giving immediate time context without hovering over the graph.

Currently the X-axis is completely hidden for minimalism, but an optional weekday label mode could balance clarity and simplicity:

**Examples:**
- If `hours_to_show: 168` (7 days), show day initials `Mon`, `Tue`, etc.
- Optional: support `show_weekday_labels: true/false`.

**Why?**
- Provides quick context for 7‑day views.
- Enhances usability without cluttering the design.

Would love to hear your thoughts or collaborate on this feature 😊

Cheers,

**Comment by Robster64** [comment]
Yes please!  One more vote for x-axis value labels.  It’s a surprising omission on an otherwise-perfect card.  Takes away much of the ability to interpret the data.  Very grateful for all the work done to-date on the card!
